src/app/store/config/selectors.ts

```diff
@@ -215,6 +215,15 @@ const theme = createSelector([all], (configs) =>
   getValueFromName<string>(configs, ConfigNames.THEME)
 );
 
+/**
+ * Returns the MAAS config for session length.
+ * @param - The redux state
+ * @returns The MAAS session length.
```

Comment:
Might be worth explaining what's the returned time unit.

Reply:
Sure, I'll extend the comment